src/main/java/org/reaktivity/nukleus/tcp/internal/connector/Connector.java

```diff
@@ -120,7 +120,6 @@ private int processConnect(
         catch (Exception ex)
         {
             handleConnectFailed(request);
-            LangUtil.rethrowUnchecked(ex);
```

Comment:
Excellent - this is not a real failure scenario for the TCP Nukleus, just the connection.